hack/make/ubuntu

```diff
@@ -150,6 +160,7 @@ EOF
 			--deb-recommends ca-certificates \
 			--deb-recommends git \
 			--deb-recommends xz-utils \
+			--deb-recommends apparmor \
```

Comment:
In this package, this should really only be a `Suggests` relation (not quite as strong as `Recommends`).

https://www.debian.org/doc/debian-policy/ch-relationships.html#s-binarydeps

> **Recommends**
> This declares a strong, but not absolute, dependency.
> 
> The Recommends field should list packages that would be found together with this one in all but unusual installations.
> 
> **Suggests**
> This is used to declare that one package may be more useful with one or more others. Using this field tells the packaging system and the user that the listed packages are related to this one and can perhaps enhance its usefulness, but that installing this one without them is perfectly reasonable.